src/theory/arith/arith_poly_norm.cpp

```diff
@@ -381,6 +381,16 @@ PolyNorm PolyNorm::mkPolyNorm(TNode n)
         }
         continue;
       }
+      else if (k==Kind::DIVISION || k == Kind::DIVISION_TOTAL)
+      {
+        // only division by non-zero constant is supported
+        if (cur[1].isConst() && cur[1].getConst<Rational>().sgn()!=0)
```

Comment:
If this is not enough, it's also fine if you consider expressions containing only constants.

Reply:
I'd prefer not to go that direction, these should definitely be normalized by this point.